### PR TITLE
5.2 update

### PR DESCRIPTION
**Pull request by attickdoor**

### CODE REVIEW

File: plugins/MOAction/MOAction.json [hunk 1]
```diff
@@ -3,5 +3,5 @@
 	"Name": "Mouseover Action",
 	"Description": "This plugin allows for actions to be used on mouseover targets without a macro.",
 	"InternalName": "MOAction",
-	"AssemblyVersion": "1.0.0.0"
+	"AssemblyVersion": "1.0.0.1"
```
Comment (goaaats) [inline plugins/MOAction/MOAction.json]: Can you add the ApplicableGameVersion in your source files?

File: plugins/XIVCombo/XIVCombo.json [hunk 2]
```diff
@@ -3,5 +3,5 @@
 	"Name": "XIV Combo",
 	"Description": "This plugin condenses combos and mutually exclusive abilities onto a single button.",
 	"InternalName": "XIVCombo",
-	"AssemblyVersion": "1.0.0.0"
+	"AssemblyVersion": "1.0.0.1"
```
Comment (goaaats) [inline plugins/XIVCombo/XIVCombo.json]: Same thing here - thanks a lot! I'll update my samples asap!

File: plugins/MOAction/MOAction.json (hunk 1 above)
Comment (attickdoor) [reply plugins/MOAction/MOAction.json]: Added.

File: plugins/XIVCombo/XIVCombo.json (hunk 2 above)
Comment (attickdoor) [reply plugins/XIVCombo/XIVCombo.json]: Should be good to go now, I think?

**Pull request merged**